spec/integration/core/vapp_spec.rb

```diff
@@ -174,7 +174,7 @@
           "invalid-vapp-template-id",
           @test_params.vdc_1_name
         )
-      }.to raise_error(Fog::Compute::VcloudDirector::Forbidden, "This operation is denied.")
+      }.to raise_error(Fog::Compute::VcloudDirector::Forbidden, /\[ #{uuid_matcher} \] This operation is denied/)
```

Comment:
@mattbostock thats exactly what I had originally done, but changed as we have and use a uuid_matcher elsewhere too so I thought I'd keep in line.